handler.go

```diff
@@ -0,0 +1,17 @@
+package otelutil
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func WithSpanStartAttrs(attrs ...attribute.KeyValue) otelhttp.Option {
```

Comment:
it would be good to expose KeyValue type in trace.go file and ultimately change the code to be like this

```suggestion
func WithSpanStartAttrs(attrs ...KeyValue) otelhttp.Option {
```

Reply:
I have tried this; I found it introduces more complexity. Considering the library exposes `var AttrString = attribute.String`, i think by using `AttrString` is much easier to write the code